src/Notification.ts

```diff
@@ -8,7 +8,7 @@ export class Notification<T> {
     this.hasValue = kind === 'N';
   }
 
-  observe(observer: Observer<T>): any {
+  observe(observer: Observer<T>): void {
     switch (this.kind) {
       case 'N':
         return observer.next(this.value);
```

Comment:
actually I was thinking this also, is it necessarily have to return here while `Observer::next` (or error, complete) doesn't return anything? what about also refactor this at once?